tests/cupyx_tests/scipy_tests/sparse_tests/test_dia.py

```diff
@@ -291,12 +291,12 @@ def test_transpose(self, xp, sp):
         m = self.make(xp, sp, self.dtype)
         return m.transpose()
 
-    @testing.with_requires('scipy>=1.0')
+    @testing.with_requires('scipy>=1.5.0')
```

Comment:
Could you add some comments for this restriction? (Just a link to this PR would be fine!)

Reply:
Added comment!

Reply:
rebased & force pushed